tools/build_defs/shell_toolchain/toolchains/impl/osx_commands.bzl

```diff
@@ -106,17 +106,13 @@ fi
 def symlink_to_dir(source, target):
     text = """local target="$2"
 mkdir -p "$target"
-if [[ -f "$1" ]]; then
+if [[ -d "$1" ]]; then
```

Comment:
I needed to revert to previous otherwise it doesn't work for me

Reply:
Sorry, I don't think I understand what you're referring to here. Can you elaborate a bit?